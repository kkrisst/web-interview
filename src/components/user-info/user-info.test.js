import React from 'react'
import { shallow } from 'enzyme'

import UserInfo from './user-info.component.jsx'

it('expect to render UserInfo component', () => {
  expect(shallow(<UserInfo />)).toMatchSnapshot()
})

it('componentDidMount fetched corretly', async () => {
  const wrapper = shallow(<UserInfo userId="1" />)
  await wrapper.instance().fetchUserInfo()
  const state = wrapper.instance().state
  console.log(state)
})

it('componentDidMount calls fetchUserInfo', async () => {
  const wrapper = shallow(<UserInfo userId="1" />)
  await wrapper.instance().componentDidMount()
  wrapper.update()
  //expect(wrapper.instance().bar).toBe(100);
})
