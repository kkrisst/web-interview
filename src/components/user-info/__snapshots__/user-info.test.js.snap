// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render UserInfo component 1`] = `ShallowWrapper {}`;
